lib/datasource.js

```diff
@@ -456,11 +456,21 @@ DataSource.prototype.defineRelations = function(modelClass, relations) {
     }
   };
 
+  var hasInvalidRelationName = function(relationName) {
+    var invalidRelationNames = ['trigger'];
+
+    return invalidRelationNames.some(function(element) {
```

Comment:
Since we are doing a simple value-comparison here, `Array.prototype.indexOf` is sufficient.